Resources/Prototypes/Entities/Objects/Misc/space_cash.yml

```diff
@@ -9,6 +9,9 @@
     shape:
     - 0,0,1,0
     storedOffset: 0,-2
+  - type: Currency
+    price:
+      Speso: 1
```

Comment:
```suggestion
```

This is the only issue I can see - for what it's worth, if you aren't using a Currency, you could consider whitelisting the Cash component inside mail capsules.

Also, might want to take the recoloured 100k sprite.